ui/app/styles/registration/_patientCommonInformation.scss

```diff
@@ -9,6 +9,12 @@
       display:inline-block;
       margin:8px 5px 8px 5px;
     }
+    &.uneditable-input{
+      width: 5.7em;
+    }
+  }
+  .pre-patient-label{
+    width: 14em !important;
```

Comment:
@kavitha-sundararajan to be consistent it would be great if this naming also can be changed too

Reply:
updated as above @deepthi-mantena